types/oracledb/index.d.ts

```diff
@@ -458,6 +503,14 @@ declare namespace OracleDB {
     const SHUTDOWN_MODE_TRANSACTIONAL: number;
     /** Constant for shutting down the Oracle database with oracledb.shutdown() and connection.shutdown() */
     const SHUTDOWN_MODE_TRANSACTIONAL_LOCAL: number;
+    /** Constants for the vectorFormat attribute. */
+    const VECTOR_FORMAT_BINARY: number;
+    /** Constants for the vectorFormat attribute. */
```

Comment:
You can keep seperate comments like indicates FLOAT32 storage format.

Reply:
Following the standard used with other constants.